### PR TITLE
Add example of multiple ports to documentation

### PR DESCRIPTION
**Pull request by nosvalds**

### CODE REVIEW

**Comment by pulkitaggarwl** [comment]
@kanika1894 can you confirm if the Workflow file will work after the suggested changes.

File: README.md [hunk 1]
```diff
@@ -86,6 +86,9 @@ jobs:
             registry-password: ${{ secrets.REGISTRY_PASSWORD }}
             name: contoso-container
             location: 'west us'
+            ports:
+              80
```
Comment (kanika1894) [inline README.md:90]: https://github.com/Azure/aci-deploy/blob/master/action.yml#L96
We can have them separated by space instead of a new line.

Comment (kanika1894) [reply README.md:90]: @nosvalds can you please update this?

Comment (nosvalds) [reply README.md:90]: Yes. I've made the fix on my Fork but for some reason this PR thinks my Fork is deleted:

https://github.com/IATI/aci-deploy/tree/port-documentation

image

I might need to make a new PR...

Comment (kanika1894) [reply README.md:90]: ohh, that's strange
So @nosvalds, planning to create a new PR then?

**Comment by kanika1894** [comment]
As you plan to create a new PR, closing this one.
@nosvalds, looking forward to your contribution. Thanks!

**Pull request closed without merging**